README.md

```diff
@@ -123,7 +123,7 @@ HOME_RPC_URL=https://sokol.poa.network
 FOREIGN_RPC_URL=https://kovan.infura.io/mew
 HOME_BRIDGE_ADDRESS=0xABb4C1399DcC28FBa3Beb76CAE2b50Be3e087353
 FOREIGN_BRIDGE_ADDRESS=0xE405F6872cE38a7a4Ff63DcF946236D458c2ca3a
-POA20_ADDRESS=0x6F794fb14d01f7551C1fe5614FDDf5895A2e82d3
+ERC20_ADDRESS=0x6F794fb14d01f7551C1fe5614FDDf5895A2e82d3
```

Comment:
Can this value be extracted from the bridge contract?

Reply:
It can be extracted. You want me to make the change?

Reply:
yes, please

Reply:
Done in https://github.com/poanetwork/bridge-monitor/pull/12/commits/e8fc5b7b5f0c3d16a037617089b5c920304ed849

Reply:
So it could be removed now, right?